specification/resources/databases/databases_update_installUpdate.yml

```
@@ -0,0 +1,47 @@
+operationId: databases_update_installUpdate
+
+summary: Start Database Maintenance
+
+description: >-
+  To start the installation of updates now for a
+  database cluster, send a PUT request to
```

```suggestion
  To start the installation of updates for a
  database cluster, send a PUT request to
```
I think removing the "now" might be easier to read?